app/lib/pages/home/firmware_update.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
```

Comment:
Did you test this with the compiled android and ios versions? I've found issues before with the nrf library

Reply:
Yes I buit an APK and tested as well